### ISSUE TITLE
A wrong alert definition is causing kapacitor to crash and make it impossible to restart.

### ISSUE THREAD

**Issue by AlexGrs** [influxdata/kapacitor#297]
Alert definition:
// Get success

```
var success = batch
                .query('''SELECT sum(value) FROM "metrics_data"."default"."c_test_status" WHERE status='success' ''')
                .period(5m)
                .every(10s)
                .groupBy(time(1m))
                .fill(0)

var failed = batch
                .query(''' SELECT sum(value) FROM "metrics_data"."default"."c_test_status" WHERE status='failed' ''')
                .period(5m)
                .every(10s)
                .groupBy(time(1m))
                .fill(0)

success.join(failed)
        .as('test_success','test_failed')
        .eval(lambda: "test_failed.sum" / ("test_success.sum" + "test_failed.sum")).as('value')
        .groupBy(time(1m))
        .influxDBOut()
            .database('test')
            .measurement('error_percent')
```

Kapacitor output:

```
2016/03/09 11:47:52 Using configuration at: kapacitor.conf
[run] 2016/03/09 11:47:52 I! Kapacitor starting, version 0.10.1, branch master, commit 22b917b4722addf9ff604d16fa01a09431641c55
[run] 2016/03/09 11:47:52 I! Go version go1.5.3, GOMAXPROCS set to 12
[srv] 2016/03/09 11:47:52 I! Kapacitor hostname: localhost
[task_master] 2016/03/09 11:47:52 I! opened
[srv] 2016/03/09 11:47:52 I! ClusterID: 11b1b702-aed3-45d1-8682-c4cdbd3e654e ServerID: 415bc9d3-bd2b-4f85-9c07-11e2809f9731
[httpd] 2016/03/09 11:47:52 I! Starting HTTP service
[httpd] 2016/03/09 11:47:52 I! Authentication enabled: false
[httpd] 2016/03/09 11:47:52 I! Listening on HTTP: [::]:9092
[influxdb] 2016/03/09 11:47:52 I! linking subscriptions
[udp:metrics_data.default] 2016/03/09 11:47:52 I! Started listening on UDP: 127.0.0.1:54563
[influxdb] 2016/03/09 11:47:52 I! started UDP listener for metrics_data default
[udp:_internal.monitor] 2016/03/09 11:47:52 I! Started listening on UDP: 127.0.0.1:46974
[influxdb] 2016/03/09 11:47:52 I! started UDP listener for _internal monitor
[udp:test.default] 2016/03/09 11:47:52 I! Started listening on UDP: 127.0.0.1:36009
[influxdb] 2016/03/09 11:47:52 I! started UDP listener for test default
[task_master] 2016/03/09 11:47:52 I! Started task: rtes
[stats] 2016/03/09 11:47:52 I! opened service
panic: not implemented

goroutine 67 [running]:
github.com/influxdata/kapacitor.(*GroupByNode).runGroupBy(0xc8201b8c40, 0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/group_by.go:44 +0x360
github.com/influxdata/kapacitor.(*GroupByNode).(github.com/influxdata/kapacitor.runGroupBy)-fm(0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/group_by.go:25 +0x4c
github.com/influxdata/kapacitor.(*node).start.func1(0xc8201b8c40, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:95 +0x7f
created by github.com/influxdata/kapacitor.(*node).start
	/root/go/src/github.com/influxdata/kapacitor/node.go:96 +0x8d

goroutine 1 [runnable]:
unicode/utf8.EncodeRune(0xc8201453b8, 0x4, 0x4, 0xc800000064, 0x1)
	/usr/local/go/src/unicode/utf8/utf8.go:340
strconv.quoteWith(0xc82017369d, 0x7, 0x570022, 0x0, 0x0)
	/usr/local/go/src/strconv/quote.go:42 +0x1292
strconv.Quote(0xc820173680, 0x24, 0x0, 0x0)
	/usr/local/go/src/strconv/quote.go:93 +0x41
expvar.(*String).String(0x10414e0, 0x0, 0x0)
	/usr/local/go/src/expvar/expvar.go:222 +0x8f
github.com/influxdata/kapacitor.GetStringVar(0xc52480, 0xa, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/global_stats.go:51 +0x62
github.com/influxdata/kapacitor/services/reporting.(*Service).Open(0xc8200ac7e0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/services/reporting/service.go:50 +0x8a
github.com/influxdata/kapacitor/cmd/kapacitord/run.(*Server).Open.func1(0xc8200c1f00, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/cmd/kapacitord/run/server.go:428 +0x4e3
github.com/influxdata/kapacitor/cmd/kapacitord/run.(*Server).Open(0xc8200c1f00, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/cmd/kapacitord/run/server.go:408 +0x31
github.com/influxdata/kapacitor/cmd/kapacitord/run.(*Command).Run(0xc8200ac510, 0xc82007e160, 0x2, 0x2, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/cmd/kapacitord/run/command.go:121 +0x11a7
main.(*Main).Run(0xc820145ef0, 0xc82007e160, 0x2, 0x2, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/cmd/kapacitord/main.go:80 +0x678
main.main()
	/root/go/src/github.com/influxdata/kapacitor/cmd/kapacitord/main.go:41 +0x363

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 20 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 6 [select]:
github.com/influxdata/kapacitor.(*Edge).NextBatch(0xc8202a3e00, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:130 +0x18a
github.com/influxdata/kapacitor.(*Edge).Next(0xc8202a3e00, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:115 +0xee
github.com/influxdata/kapacitor.(*JoinNode).runJoin.func1(0xc8200104b0, 0xc820202300, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/join.go:79 +0x9b
created by github.com/influxdata/kapacitor.(*JoinNode).runJoin
	/root/go/src/github.com/influxdata/kapacitor/join.go:87 +0x12f

goroutine 23 [select]:
github.com/influxdata/kapacitor.(*Edge).NextPoint(0xc820173170, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:119 +0x17d
github.com/influxdata/kapacitor.(*TaskMaster).runForking(0xc82007d600, 0xc820173170)
	/root/go/src/github.com/influxdata/kapacitor/task_master.go:380 +0x76
created by github.com/influxdata/kapacitor.(*TaskMaster).stream
	/root/go/src/github.com/influxdata/kapacitor/task_master.go:374 +0x14f

goroutine 40 [chan receive]:
github.com/influxdata/kapacitor/services/udp.(*Service).processPackets(0xc8201ec280)
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:143 +0xe6
created by github.com/influxdata/kapacitor/services/udp.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:105 +0xc84

goroutine 27 [IO wait]:
net.runtime_pollWait(0x7f8816cf6848, 0x72, 0xc82007a0a0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820136b50, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820136b50, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).accept(0xc820136af0, 0x0, 0x7f8816cf6988, 0xc820183340)
	/usr/local/go/src/net/fd_unix.go:408 +0x27c
net.(*TCPListener).AcceptTCP(0xc820086120, 0x455ad0, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
net.(*TCPListener).Accept(0xc820086120, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:264 +0x3d
net/http.(*Server).Serve(0xc820078c00, 0x7f8816cf58c8, 0xc820086120, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:1887 +0xb3
net/http.Serve(0x7f8816cf58c8, 0xc820086120, 0x7f8816cf6960, 0xc8200789c0, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:1765 +0x85
github.com/influxdata/kapacitor/services/httpd.(*Service).serve.func1(0xc820089770)
	/root/go/src/github.com/influxdata/kapacitor/services/httpd/service.go:114 +0x6e
created by github.com/influxdata/kapacitor/services/httpd.(*Service).serve
	/root/go/src/github.com/influxdata/kapacitor/services/httpd/service.go:122 +0x35

goroutine 39 [IO wait]:
net.runtime_pollWait(0x7f8816cf6788, 0x72, 0xc82007a0a0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8201b4760, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8201b4760, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).readFrom(0xc8201b4700, 0xc820274000, 0x10000, 0x10000, 0x0, 0x0, 0x0, 0x7f8816cef028, 0xc82007a0a0)
	/usr/local/go/src/net/fd_unix.go:259 +0x2a5
net.(*UDPConn).ReadFromUDP(0xc8201ae028, 0xc820274000, 0x10000, 0x10000, 0x10000, 0x10000, 0x0, 0x0)
	/usr/local/go/src/net/udpsock_posix.go:61 +0x117
github.com/influxdata/kapacitor/services/udp.(*Service).serve(0xc8201ec280)
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:125 +0x166
created by github.com/influxdata/kapacitor/services/udp.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:103 +0xc39

goroutine 7 [select]:
github.com/influxdata/kapacitor.(*Edge).NextBatch(0xc8202ce030, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:130 +0x18a
github.com/influxdata/kapacitor.(*Edge).Next(0xc8202ce030, 0x0, 0x0, 0x180001)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:115 +0xee
github.com/influxdata/kapacitor.(*JoinNode).runJoin.func1(0xc8200104b0, 0xc820202300, 0x1)
	/root/go/src/github.com/influxdata/kapacitor/join.go:79 +0x9b
created by github.com/influxdata/kapacitor.(*JoinNode).runJoin
	/root/go/src/github.com/influxdata/kapacitor/join.go:87 +0x12f

goroutine 51 [select]:
github.com/influxdata/kapacitor.(*Edge).NextBatch(0xc8202ce8d0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:130 +0x18a
github.com/influxdata/kapacitor.(*BatchNode).runBatch.func1(0xc820079980, 0xc820202100)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:297 +0xa5
created by github.com/influxdata/kapacitor.(*BatchNode).runBatch
	/root/go/src/github.com/influxdata/kapacitor/batch.go:307 +0x81

goroutine 41 [IO wait]:
net.runtime_pollWait(0x7f8816cf6608, 0x72, 0xc82007a0a0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8201b47d0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8201b47d0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).readFrom(0xc8201b4770, 0xc82025c000, 0x10000, 0x10000, 0x0, 0x0, 0x0, 0x7f8816cef028, 0xc82007a0a0)
	/usr/local/go/src/net/fd_unix.go:259 +0x2a5
net.(*UDPConn).ReadFromUDP(0xc8201ae030, 0xc82025c000, 0x10000, 0x10000, 0x10000, 0x10000, 0x0, 0x0)
	/usr/local/go/src/net/udpsock_posix.go:61 +0x117
github.com/influxdata/kapacitor/services/udp.(*Service).serve(0xc8201ec320)
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:125 +0x166
created by github.com/influxdata/kapacitor/services/udp.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:103 +0xc39

goroutine 42 [chan receive]:
github.com/influxdata/kapacitor/services/udp.(*Service).processPackets(0xc8201ec320)
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:143 +0xe6
created by github.com/influxdata/kapacitor/services/udp.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:105 +0xc84

goroutine 43 [IO wait]:
net.runtime_pollWait(0x7f8816cf66c8, 0x72, 0xc82007a0a0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8201b4840, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8201b4840, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).readFrom(0xc8201b47e0, 0xc820284000, 0x10000, 0x10000, 0x0, 0x0, 0x0, 0x7f8816cef028, 0xc82007a0a0)
	/usr/local/go/src/net/fd_unix.go:259 +0x2a5
net.(*UDPConn).ReadFromUDP(0xc8201ae038, 0xc820284000, 0x10000, 0x10000, 0x10000, 0x10000, 0x0, 0x0)
	/usr/local/go/src/net/udpsock_posix.go:61 +0x117
github.com/influxdata/kapacitor/services/udp.(*Service).serve(0xc8201ec3c0)
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:125 +0x166
created by github.com/influxdata/kapacitor/services/udp.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:103 +0xc39

goroutine 44 [chan receive]:
github.com/influxdata/kapacitor/services/udp.(*Service).processPackets(0xc8201ec3c0)
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:143 +0xe6
created by github.com/influxdata/kapacitor/services/udp.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/udp/service.go:105 +0xc84

goroutine 47 [select]:
github.com/influxdata/kapacitor.(*BatchNode).runBatch(0xc820202100, 0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:312 +0x4ce
github.com/influxdata/kapacitor.(*BatchNode).(github.com/influxdata/kapacitor.runBatch)-fm(0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:120 +0x4c
github.com/influxdata/kapacitor.(*node).start.func1(0xc820202100, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:95 +0x7f
created by github.com/influxdata/kapacitor.(*node).start
	/root/go/src/github.com/influxdata/kapacitor/node.go:96 +0x8d

goroutine 48 [select]:
github.com/influxdata/kapacitor.(*BatchNode).runBatch(0xc820202200, 0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:312 +0x4ce
github.com/influxdata/kapacitor.(*BatchNode).(github.com/influxdata/kapacitor.runBatch)-fm(0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:120 +0x4c
github.com/influxdata/kapacitor.(*node).start.func1(0xc820202200, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:95 +0x7f
created by github.com/influxdata/kapacitor.(*node).start
	/root/go/src/github.com/influxdata/kapacitor/node.go:96 +0x8d

goroutine 49 [semacquire]:
sync.runtime_Semacquire(0xc8200104bc)
	/usr/local/go/src/runtime/sema.go:43 +0x26
sync.(*WaitGroup).Wait(0xc8200104b0)
	/usr/local/go/src/sync/waitgroup.go:126 +0xb4
github.com/influxdata/kapacitor.(*JoinNode).runJoin(0xc820202300, 0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/join.go:89 +0x14f
github.com/influxdata/kapacitor.(*JoinNode).(github.com/influxdata/kapacitor.runJoin)-fm(0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/join.go:64 +0x4c
github.com/influxdata/kapacitor.(*node).start.func1(0xc820202300, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:95 +0x7f
created by github.com/influxdata/kapacitor.(*node).start
	/root/go/src/github.com/influxdata/kapacitor/node.go:96 +0x8d

goroutine 66 [select]:
github.com/influxdata/kapacitor.(*Edge).NextBatch(0xc8202ce270, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:130 +0x18a
github.com/influxdata/kapacitor.(*EvalNode).runEval(0xc8201b8b60, 0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/eval.go:55 +0x380
github.com/influxdata/kapacitor.(*EvalNode).(github.com/influxdata/kapacitor.runEval)-fm(0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/eval.go:34 +0x4c
github.com/influxdata/kapacitor.(*node).start.func1(0xc8201b8b60, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:95 +0x7f
created by github.com/influxdata/kapacitor.(*node).start
	/root/go/src/github.com/influxdata/kapacitor/node.go:96 +0x8d

goroutine 68 [runnable]:
github.com/influxdata/kapacitor.(*Edge).NextBatch(0xc8202ce690, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:130 +0x18a
github.com/influxdata/kapacitor.(*InfluxDBOutNode).runOut(0xc8201bcb60, 0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/influxdb_out.go:42 +0x4a8
github.com/influxdata/kapacitor.(*InfluxDBOutNode).(github.com/influxdata/kapacitor.runOut)-fm(0x0, 0x0, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/influxdb_out.go:22 +0x4c
github.com/influxdata/kapacitor.(*node).start.func1(0xc8201bcb60, 0x0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:95 +0x7f
created by github.com/influxdata/kapacitor.(*node).start
	/root/go/src/github.com/influxdata/kapacitor/node.go:96 +0x8d

goroutine 69 [select]:
github.com/influxdata/kapacitor.(*ExecutingTask).runSnapshotter(0xc8202b60e0)
	/root/go/src/github.com/influxdata/kapacitor/task.go:373 +0x8a2
created by github.com/influxdata/kapacitor.(*ExecutingTask).start
	/root/go/src/github.com/influxdata/kapacitor/task.go:184 +0x236

goroutine 70 [select]:
github.com/influxdata/kapacitor.(*BatchNode).doQuery(0xc820202100, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:241 +0xd2e
github.com/influxdata/kapacitor.(*BatchNode).Start.func1(0xc820202100)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:198 +0x21
created by github.com/influxdata/kapacitor.(*BatchNode).Start
	/root/go/src/github.com/influxdata/kapacitor/batch.go:199 +0xd3

goroutine 71 [select]:
github.com/influxdata/kapacitor.(*BatchNode).doQuery(0xc820202200, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:241 +0xd2e
github.com/influxdata/kapacitor.(*BatchNode).Start.func1(0xc820202200)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:198 +0x21
created by github.com/influxdata/kapacitor.(*BatchNode).Start
	/root/go/src/github.com/influxdata/kapacitor/batch.go:199 +0xd3

goroutine 72 [chan receive]:
github.com/influxdata/kapacitor.(*node).Err(0xc8201bcb60, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/node.go:117 +0xc6
github.com/influxdata/kapacitor.(*ExecutingTask).Err.func1(0x7f8816cb46c8, 0xc8201bcb60, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/task.go:268 +0x33
github.com/influxdata/kapacitor.(*ExecutingTask).rwalk(0xc8202b60e0, 0xda50f0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/task.go:112 +0x91
github.com/influxdata/kapacitor.(*ExecutingTask).Err(0xc8202b60e0, 0x0, 0x0)
	/root/go/src/github.com/influxdata/kapacitor/task.go:269 +0x39
github.com/influxdata/kapacitor/services/task_store.(*Service).StartTask.func1(0xc8202b60e0, 0xc820078a20, 0xc82019f340)
	/root/go/src/github.com/influxdata/kapacitor/services/task_store/service.go:613 +0x30
created by github.com/influxdata/kapacitor/services/task_store.(*Service).StartTask
	/root/go/src/github.com/influxdata/kapacitor/services/task_store/service.go:625 +0x271

goroutine 82 [select]:
github.com/influxdata/kapacitor.(*Edge).NextBatch(0xc8202ceae0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:130 +0x18a
github.com/influxdata/kapacitor.(*BatchNode).runBatch.func1(0xc8202dc000, 0xc820202200)
	/root/go/src/github.com/influxdata/kapacitor/batch.go:297 +0xa5
created by github.com/influxdata/kapacitor.(*BatchNode).runBatch
	/root/go/src/github.com/influxdata/kapacitor/batch.go:307 +0x81

goroutine 73 [runnable]:
github.com/influxdata/kapacitor.(*Edge).NextPoint(0xc8202cee70, 0xda5e60, 0xc82007d73c, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/root/go/src/github.com/influxdata/kapacitor/edge.go:118
github.com/influxdata/kapacitor.(*TaskMaster).runForking(0xc82007d600, 0xc8202cee70)
	/root/go/src/github.com/influxdata/kapacitor/task_master.go:380 +0x76
created by github.com/influxdata/kapacitor.(*TaskMaster).stream
	/root/go/src/github.com/influxdata/kapacitor/task_master.go:374 +0x14f

goroutine 74 [select]:
github.com/influxdata/kapacitor/services/stats.(*Service).sendStats(0xc8200b9e00)
	/root/go/src/github.com/influxdata/kapacitor/services/stats/service.go:100 +0x179
created by github.com/influxdata/kapacitor/services/stats.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/stats/service.go:76 +0x1fa
```

**Comment by AlexGrs** [comment]
Ok so i was able to restart kapacitor by going in the task definition folder and by deleting the database.

**Comment by nathanielc** [comment]
@AlexGrs Thank for reporting the issue and I am glad you found the workaround. The `not implemented` panic has been fixed #236 in the master branch. Also panics will no longer be used to indicate errors so that the task can be modified to fix the error.
